modules/crio_get.adoc

```diff
@@ -0,0 +1,117 @@
+= Getting CRI-O
+CRI-O is not supported as a stand-alone container engine.
```

Comment:
Does this paragraph belong here? By now, the user has decided he/she wants CRI-O enough to get it ?
Maybe move to _*modules/crio_arch.adoc*_

Reply:
I'm going to leave it. An intro is needed to the next section.